invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -53,7 +56,11 @@ class Meta:
         values=fields.Nested(PIDSchema),
     )
     metadata = NestedAttribute(MetadataSchema)
-    # ext = fields.Method('dump_extensions', 'load_extensions')
+    # FIXME: with NestedAttribute it does not work
```

Comment:
Not clear what should be fixed here? can we fix before merge?

Reply:
Actually nothing. It was fixed but the FIXME not removed. Nice catch!